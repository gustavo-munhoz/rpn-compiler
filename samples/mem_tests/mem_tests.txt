(10 MEM)
(MEM)
((MEM) 5 +)
(-2.5 MEM)
(MEM)
((MEM) (MEM) *)