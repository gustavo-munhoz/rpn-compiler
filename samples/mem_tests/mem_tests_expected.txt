(10 MEM)                 => 10 (0x4900)
(MEM)                    => 10 (0x4900)
((MEM) 5 +)              => 15 (0x4B80)
(-2.5 MEM)               => -2.5 (0xC100)
(MEM)                    => -2.5 (0xC100)
((MEM) (MEM) *)          => 6.25 (0x4640)